docker-image/gpu/spark1.6.3/Dockerfile

```diff
@@ -1,4 +1,159 @@
-FROM  aztk/spark:v0.1.0-spark1.6.3-base
```

Comment:
why not build on top of this base image any more?

Reply:
There is a difference between base and gpu images in the Spark building process now (the added -Pnetlib-lgpl). We could build all images with that profile -- that might be the best option here.